scripts/update_terminfo.sh

```diff
@@ -61,8 +61,13 @@ cat > "$target" <<EOF
 // it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
```

Comment:
I also noticed that none of our headers have this PVS message. I guess it's not needed. Right @ZyX-I ?

Reply:
PVS (currently) operates on C files only, it does not analyze header files (side from the fact that PVS runs preprocessor first which dumps used headers inside C files) and thus does not need headers there.